packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.18+4
+
+* Uses Text.rich to replace RichText so the widget can work with `SelectionArea` when `selectable` set to false.
```

Comment:
Please use backticks consistently for all code references.

Reply:
Done.